src/app/business/routes/objective/components/detail/components/tabs/index.js

```diff
@@ -3,19 +3,19 @@ import PropTypes from 'prop-types';
 import {PulseLoader} from 'react-spinners';
 import {noop} from 'lodash';
 import styled from '@emotion/styled';
-import {
-    TabList,
-    Tabs,
-    TabPanel,
-    RoundedButton,
-    DownloadSimple,
-} from '@substrafoundation/substra-ui';
 
-import Tab from '../../../../../../common/components/detail/components/tabs';
 import PermissionsTabPanelContent from '../../../../../../common/components/detail/components/permissionsTabPanelContent';
 import ForbiddenResource from '../../../../../../common/components/detail/components/forbiddenResource';
 import Description from '../../../../../../common/components/detail/components/description';
 import {spacingNormal} from '../../../../../../../../../assets/css/variables/spacing';
+import {
+    TabList,
+    Tab,
```

Comment:
This was actually imported from `common/components/detail/components/tabs` beforehand. Could you check what the difference is between the two components and decide which one should be used (and if they are both needed?)

Reply:
It looks like the `common/components/detail/components/tabs` was a layer added to SubstraUI's Tab component that enabled us to change the tab's color following the front-end we were using. This file isn't used anymore, we could even remove it.